vocab/credentials.jsonld

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
Here's the latest context from the test suite: https://github.com/w3c/vc-test-suite/blob/gh-pages/contexts/vc-v1.jsonld